src/server/index.js

```diff
@@ -23,10 +23,11 @@ app.use(express.json({
 const router = express.Router()
 app.use(routes(router))
 
+app.use(express.static('dist'))
+app.use(fallback('index.html', { root: path.join(__dirname, '../../dist') }))
+
```

Comment:
It is important to have this ordering so that the server routes get hit before the frontend as Winggo found out... :)